src/filters/colormatrix/ColorMatrixFilter.js

```diff
@@ -575,6 +577,26 @@ export default class ColorMatrixFilter extends core.Filter
     {
         this.uniforms.m = value;
     }
+
+    /**
+     * The interpolation value to use when mixing the original and resultant colors.
+     *
+     * When the value is 0, the original color is used without modification.
+     * When the value is 1, the result color is used.
+     * When in the range (0, 1) the color is interpolated between the original and result by this amount.
+     *
+     * @member {number}
+     * @default 1
+     */
+    get interpolation()
```

Comment:
Other thought, what about "strength" it's used on some of the other filters.

amount is fine, just a little generic. I agree that alpha and blend are problematic.